requirements.txt

```diff
@@ -1,18 +1,23 @@
-Flask==1.1.2
-flask_httpauth
+
+Flask==2.1.2
+Flask-HTTPAuth==4.7.0
```

Comment:
Do these need to be pinned to exact versions, rather than `>=` or `~=`?

Reply:
:+1: